records_mover/records/targets/table/move_from_temp_loc_after_filling_it.py

```diff
@@ -28,7 +28,13 @@ def __init__(self,
     @contextmanager
     def temporary_loadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
         driver = self.tbl.db_driver(self.tbl.db_engine)
-        with driver.temporary_loadable_directory_loc() as loc:
+        loader = driver.loader()
+        # This will only be reached in move() if
+        # Source#has_compatible_format(records_target) returns true,
+        # which means we were able to get a laoder and call
+        # can_load_this_format() previously.
+        assert loader is not None
+        with loader.temporary_loadable_directory_loc() as loc:
```

Comment:
These assertions are an example of making the table source/target a little more complex in order to make the driver simpler.